conf/spark/spark-defaults.conf

```diff
@@ -3,7 +3,7 @@ spark.eventLog.dir                 file:///home/mnt/applicationHistory
 spark.history.fs.logDirectory      file:///home/mnt/applicationHistory
 spark.network.timeout              7200s
 spark.history.ui.port              18081
-spark.local.dir                    /home/vmagent/app/e2eaiok/modelzoo/dien/feature_engineering/spark_local_dir
+spark.local.dir                    /home/vmagent/app/spark_data_processing/spark_local_dir
```

Comment:
@Peach-He , want to check with you that by doing this DIR, will it use docker space instead of mounted volume? we need to make sure this spark_data_processing dir is using mounted volume.